sdk/sdktypes/program_error.go

```diff
@@ -56,11 +61,11 @@ func StrictProgramErrorFromProto(m *ProgramErrorPB) (ProgramError, error) {
 	return Strict(ProgramErrorFromProto(m))
 }
 
-func NewProgramError(msg string, callstack []CallFrame, extra map[string]string) ProgramError {
+func NewProgramError(v Value, callstack []CallFrame, extra map[string]string) ProgramError {
 	return kittehs.Must1(ProgramErrorFromProto(
```

Comment:
this should really move to the sdktypes part as sdktypes.NewProgramError(msg, extra,value,callstack) and we should stop using proto directly in backend

Reply:
not sure i follow - there is no exposing of protobuf here? it is sdktypes.Value.